docs/proposal/kmesh_support_workload.md

```diff
@@ -0,0 +1,141 @@
+
+### Address
+
+```go//
+type Address struct {
+  // Types that are assignable to Type:
+  // *Address_Workload
+  // *Address_Service
+  Type isAddress_Type `protobuf_oneof:"type"`
+}
+```
+
+### Service
+
+```go//
+type Service struct {
+  // The service name for Kubernetes, such as: "fortio-server", "Kubernetes", "istiod", "kube-dns" etc.
+  Name string `protobuf:"bytes,1,opt,name=name,proto3" json:"name,omitempty"`
+  // The namespace the service belongs to，such as: "default", "kube-system", "istio-system" etc.
+  Namespace string `protobuf:"bytes,2,opt,name=namespace,proto3" json:"namespace,omitempty"`
+  // Hostname represents the FQDN of the service.
+  // For Kubernetes，this would be <name>.<namespace>.svc.<cluster domain>, such as: "fortio-server.default.svc.cluster.local", "istiod.istio-system.svc.cluster.local" etc.
+  Hostname string `protobuf:"bytes,3,opt,name=hostname,proto3" json:"hostname,omitempty"`
+  // Address represents the addresses the service can be reached at.
+  // There may be multiple addresses for a single service if it resides in multiple networks,
+  // multiple clusters, and/or if it's dual stack (TODO: support dual stack).
+  // For a headless kubernetes service, this list will be empty.
+  Addresses []*NetworkAddress `protobuf:"bytes,4,rep,name=addresses,proto3" json:"addresses,omitempty"`
+  // Ports for the service.
+  // The target_port may be overridden on a per-workload basis.
+  Ports []*Port `protobuf:"bytes,5,rep,name=ports,proto3" json:"ports,omitempty"`
+}
+```
+
+### Workload
+
+```go
+type Workload struct {
+  // UID represents a globally unique opaque identifier for this workload, such as: "Kubernetes//Pod/default/fortio-server-deployment-59f95d774d-85nr4"
+  Uid string `protobuf:"bytes,20,opt,name=uid,proto3" json:"uid,omitempty"`
+  // Name represents the name for the workload, For Kubernetes, this is the pod name, such as: "fortio-server-deployment-59f95d774d-ljmd5"
+  Name string `protobuf:"bytes,1,opt,name=name,proto3" json:"name,omitempty"`
+  // Namespace represents the namespace for the workload. 
+  Namespace string `protobuf:"bytes,2,opt,name=namespace,proto3" json:"namespace,omitempty"`
+  // Address represents the IPv4/IPv6 address for the workload, this should be globally unique.
+  Addresses [][]byte `protobuf:"bytes,3,rep,name=addresses,proto3" json:"addresses,omitempty"`
+  // The services for which this workload is an endpoint. the key is the NamespacedHostname string of the format namespace/hostname.
+  Services map[string]*PortList `protobuf:"bytes,22,rep,name=services,proto3" json:"services,omitempty" protobuf_key:"bytes,1,opt,name=key,proto3" protobuf_val:"bytes,2,opt,name=value,proto3"`
+  // Health status of the workload，such as: "Healthy"，"Unhealthy"
+  Status WorkloadStatus `protobuf:"varint,17,opt,name=status,proto3,enum=istio.workload.WorkloadStatus" json:"status,omitempty"`
+}
+```
+
+## How kmesh subscribe and use the workload model info
```

Comment:
In the workload model, there are also related processes such as authentication and TLS, which do not seem to be mentioned in the current design. If it is just a basic process design for now, please specify this in the documentation.